app/views/devise/passwords/edit.html.erb

```diff
@@ -17,8 +17,8 @@
           </div>
 
           <div class="form-group">
-            <%= f.label :password_confirmation, "Confirm new password", class: "mb-0" %><br />
-            <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control" %>
+            <%= f.label :confirm_password, "Confirm new password", class: "mb-0" %><br />
```

Comment:
Change "Password confirmation" to "Confirm password" which is nicer to read and matches what we have elsewhere.